src/playground/service.clj

```diff
@@ -63,8 +69,8 @@
   (session-backend
    {:authfn (fn [request]
               (let [{:keys [username password]} request]
-                (when (= (session.login/password-by-username username) password)
-                  {:username username :password password})))}))
+                (when (= (session.login/password-by-username request username) (hashers/derive password))
+                  {:username username :password (hashers/derive password) :role (session.login/get-role request username)})))}))
```

Comment:
I think this whole line is redundant:

* You are looking at a `(get-in request [:session :identity])` value
* That value was injected by the Login endpoint
* Its mere presence demonstrates you are logged in. No need to call `password-by-username` or `get-role`

Reply:
do we need to delete the backend altogether?